.github/workflows/ci.yml

```diff
@@ -19,5 +19,6 @@ jobs:
     - name: Check if the git repository is clean
       run: $(exit $(git status --porcelain --untracked-files=no | head -255 | wc -l)) || (echo "Dirty git tree"; git diff; exit 1)
     - run: npm run lint
+    - run: npm run types
```

Comment:
Why don't we make it part of the lint target? We used to have Flow as part of it, having tsc there makes sense IMHO. This also helps catch bugs locally since people know `npm run lint` exists, but not the new one,